### DISCUSSION TITLE
Tasks for the weeks 11/14 - 11/26

### DISCUSSION THREAD

**Discussion by surya-prakash-susarla** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#35] [General]
Can everyone please post the changes covered during this week? Thanks!

**Comment by surya-prakash-susarla** [comment]
My update:
1. Fixed the build pipelines and onboarded them to our branch
2. Added email APIs and updated the code to send email notifications
3. Tried a few fixes for tests but there are gaps which need to be finalized and re-written before onboarding them again.

**Comment by kaushikjadhav01** [comment]
Updates:
1. Feature: Share expenses among multiple users by email address.
2. Feature: Add custom expense category.
3. Feature: Online E-commerce product searcher.
4. Feature: Phone Number verification.
5. Fix: Ratings were not being stored to Firebase.
6. Fix: Suggestions were not being stored to Firebase.
7. Feature: Alert users when overwriting Ratings & Suggestions.
8. Fix: Make google-services.json private

**Comment by vemuriPradyumna** [comment]
Updates:

- Feature: Added new Activity Class to edit existing transactions.
- Feature: New XML view file added for the edit activity class.
- Feature: Validate new and old transaction data before updating.
- Fix: Minor changes and fixes in transactions display views and classes.
